AmplifyTestCommon/Models/CustomPrimaryKey/ModelCompositePk+Schema.swift

```diff
@@ -0,0 +1,54 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// swiftlint:disable all
+import Amplify
+import Foundation
+
+extension ModelCompositePk {
```

Comment:
can we put the annotated GraphQL schema somewhere easily accessible? here or on the `struct ModelCompositePk` file, and for each of the new models you've added (ModelCompositePk, ModelCompositePkWithAssociation, etc)

Reply:
Yes, it's included in related PR https://github.com/aws-amplify/amplify-ios/pull/1811